dj_cqrs/admin.py

```diff
@@ -0,0 +1,50 @@
+from django.utils.translation import gettext_lazy
```

Comment:
Sorry, missed one thing initially. All new python files must contain copyright header. Please, check in other places.